runtime/vm/resolvesupport.cpp

```diff
@@ -1555,93 +1558,8 @@ resolveVirtualMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
 			J9UTF8 *sigUTF = J9ROMNAMEANDSIGNATURE_SIGNATURE(nameAndSig);
 			U_8* initialMethodName = J9UTF8_DATA(nameUTF);
 			U_16 initialMethodNameLength = J9UTF8_LENGTH(nameUTF);
-			BOOLEAN isVarHandle = FALSE;
 
-			switch (initialMethodNameLength) {
```

Comment:
do we still need this for the non-J9VM_OPT_OPENJDK_METHODHANDLE code?

Reply:
VM_VMHelpers::isPolymorphicVarHandleMethod has the same functionality, I opt to use the helper to reduce duplicate code in the VM